lib/config/validation.ts

```diff
@@ -96,6 +97,7 @@ export function getParentName(parentPath: string | undefined): string {
 
 export async function validateConfig(
   config: RenovateConfig,
+  isGlobalConfig: boolean,
```

Comment:
```suggestion
  isGlobalConfig: boolean,
  config: RenovateConfig,
```

Also change this order to better catch type errors